### PR TITLE
More ORCIDs and affiliations

### PR DESCRIPTION
**Pull request by stain**

### CODE REVIEW

**Comment by bedroesb** [comment]
I notice that we should homogenize the use op pipe/backslashes in the affiliations :sweat_smile:

File: _data/CONTRIBUTORS.yaml [hunk 1]
```diff
@@ -57,6 +57,8 @@ Nicola Soranzo:
 Munazah Andrabi:
     git: smza
     role: editor
+    orcid: 0000-0002-7718-5109
+    affiliation: The University of Manchester
```
Comment (nsoranzo) [inline _data/CONTRIBUTORS.yaml:61]:
```suggestion
    affiliation: The University of Manchester | ELIXIR-UK
```

**Comment by bedroesb** [comment]
thanks @stain for this! Can I merge?

**Comment by bedroesb** [comment]
Since these are not life threatening changes, I merged it for the sake of improvement :)

**Pull request merged**